src/test/ui/dyn-star/upcast.rs

```diff
@@ -1,7 +1,6 @@
-// run-pass
+// known-bug: unknown
```

Comment:
poetry

Reply:
I can file a bug for this.

Reply:
If you do, could you also add it to #102425?